constant/biz/publish.go

```diff
@@ -1,14 +1,24 @@
 package biz
 
+import "github.com/cloudwego/hertz/pkg/protocol/consts"
+
 const (
-	InvalidContentType     = 400001
-	Unable2GenerateUUID    = 500001
-	Unable2CreateThumbnail = 500002
-	Unable2UploadVideo     = 500003
-	Unable2UploadCover     = 500004
-	Unable2CreateDBEntry   = 500005
+	InvalidContentType     = 400101
```

Comment:
Why use X0010X?

Reply:
The first 3 digits represent the http status code, last 3 are IDs. Feel free to provide a better coding solution if you prefer.